scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -395,10 +383,13 @@ mod tests {
         setup_keyspaces(&session).await?;
         setup_udts(&session, &udt_schemas).await?;
         setup_tables(&session, &table_schema).await?;
-        execute_queries(&session, operations).await?;
-        replicate(&session, &table_schema.name).await?;
-        compare_changes(&session, &table_schema.name).await?;
-        compare_timestamps(&session, &table_schema).await?;
+        session.use_keyspace("test_src", false).await?;
+        for operation in operations {
+            session.query(operation.0, operation.1).await?;
+            replicate(&session, &table_schema.name).await?;
```

Comment:
It's a bit awkward because `replicate` replays all changes from the CDC log, including those caused by the previous operations. Please modify the `replicate` function so that it keeps track of the last read `cdc$time` and only processes rows with `cdc$time` > the last seen timestamp.